lib/core/utils/uuid.js

```diff
@@ -24,7 +24,7 @@ if (!_rng && _crypto && _crypto.getRandomValues) {
 }
 
 try {
-  if (!_rng && require) {
+  if (!_rng) {
```

Comment:
This change worries me. How does this change effect other supported environments?

Reply:
Before, if `require` wasn't defined, it would skip the conditional and continue on. Now, if `require` isn't defined, an error will be thrown by `require('crypto')` on the next line, which will be caught by the try/catch block. I don't think this changes the execution in a meaningful way

Reply:
Thanks for the explanation.